packages/dso-toolkit/src/styles/mixins/_grid.scss

```diff
@@ -0,0 +1,67 @@
+$dso-featured-background-color: $grasgroen-10;
```

Comment:
Wat is deze file? Hij wordt nergens geinclude, volgens mij heb je een kopie van die andere `grid.scss` gemaakt?

Reply:
file verwijderd. Een mysterie hoe die opeens daar is terecht gekomen